src/widgets/map-widget/widget.tsx

```diff
@@ -0,0 +1,102 @@
+import { useChannelLatest } from '@wuespace/telestion-client-core';
+
+import {
+	CircleMarker,
+	MapContainer,
+	TileLayer,
+	MapConsumer
+} from 'react-leaflet';
+
+import 'leaflet/dist/leaflet.css';
+import 'leaflet/dist/leaflet.js';
+import { LatLng, LatLngBounds } from 'leaflet';
+import { JsonSerializable } from '@wuespace/telestion-client-types';
+import { Switch } from '@adobe/react-spectrum';
+import { Flex } from '@adobe/react-spectrum';
+import { Heading } from '@adobe/react-spectrum';
+import { Divider } from '@adobe/react-spectrum';
+import { useState } from 'react';
+
+interface DataMessage<T extends JsonSerializable> {
+	[key: string]: JsonSerializable;
+	dataType: string;
+	result: T[];
+	className: string;
+}
+
+interface Position {
+	[key: string]: JsonSerializable;
+	satCount: number;
+	fix: number;
+	north: number;
+	east: number;
+	time: number;
+	className: string;
+}
+
+export const MONGODB_NEW =
+	'org.telestion.core.database.MongoDatabaseService/out#save';
+
+export function Widget() {
+	const latestMessage = useChannelLatest<DataMessage<Position>>(
+		MONGODB_NEW + '/de.jvpichowski.rocketsound.messages.base.GpsData'
+	);
+
+	let [expertMode, setexpertMode] = useState(false);
+
+	const boundsno = new LatLng(49.747112, 9.958076);
+	const boundssw = new LatLng(49.65585, 9.798088);
+	let bounds = new LatLngBounds(boundsno, boundssw);
+
+	if (expertMode) {
+		bounds = new LatLngBounds(new LatLng(180, 180), new LatLng(-180, -180));
+	}
+
+	return (
+		<Flex direction={'column'} height={'100%'} gap={'size-100'}>
+			<Flex
+				direction={'row'}
+				alignItems={'center'}
+				justifyContent={'space-between'}
+				marginStart={'size-100'}
+			>
+				<Heading>Map</Heading>
+				<Switch isSelected={expertMode} onChange={setexpertMode}>
```

Comment:
Forgot something, sorry:
```suggestion
				<Switch isSelected={!isBounded} onChange={setBounded}>
```

Reply:
Ahh, thats weird.
Can we rename the switch?
For example: `Bounded`

Reply:
I think I prefer a "default-off" advanced mode kind of thing (because you can just "lose" the correct map area in this mode).

While I agree that `expertMode` wasn't my brightest invention in naming things, I would like to retain the overall UI pattern, if at all possible.